scripts/oncall-email/curl.shell

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
`/bin/bash` won't pick up the version of bash installed by brew (I believe it's not installed in `/bin` where you'd find the antiquated bash 3.x shipped by Apple).